samples/document-snippets/instance.js

```diff
@@ -0,0 +1,382 @@
+    if (err) {
+      console.error('Error creating appProfile: ', err);
+      return;
+    }
+    console.log(`App-Profile created: ${appProfile.name}`);
+  });
+  // [END bigtable_create_appProfile]
+}
+
+function newTable(tableId, instanceId) {
+  // [START bigtable_create_table]
+  const instance = bigtableClient.instance(instanceId);
+
+  const options = {
+    families: ['follows'],
+  };
+
+  // You can also specify garbage collection rules for your column families.
+  // See {@link Table#createFamily} for more information about
+  // column families and garbage collection rules.
+  //-
+  // const options = {
+  //   families: [
+  //     {
+  //       name: 'follows',
+  //       rule:  {
+  //         age: {
+  //           seconds: 0,
+  //           nanos: 5000
+  //         },
+  //         versions: 3,
+  //         union: true
+  //       }
+  //     }
+  //   ]
+  // };
+
+  instance
+    .createTable(tableId, options)
+    .then(result => {
+      const newTable = result[0];
+      // const apiResponse = result[1];
+      console.log(`Table created: ${newTable.name}`);
+    })
+    .catch(err => {
+      console.error('Error creating table: ', err);
+    });
+  // [END bigtable_create_table]
+}
+
+function existsInstance(instanceId) {
+  const instance = bigtableClient.instance(instanceId);
+
+  // [START bigtable_exists_instance]
+  instance
+    .exists()
+    .then(result => {
+      const exists = result[0];
+      console.log(`Instance ${instanceId} Exists: ${exists}`);
+    })
+    .catch(err => {
+      console.error('Error in checking Instance exists: ', err);
+    });
+  // [END bigtable_exists_instance]
+}
+
+function getInstance(instanceId) {
+  const instance = bigtableClient.instance(instanceId);
+
+  // [START bigtable_get_instance]
+  instance
+    .get()
+    .then(result => {
+      const instance = result[0];
+      // const apiResponse = result[1];
+      console.log(`Instance: \n${instance}`);
+    })
+    .catch(err => {
+      console.error('Error geting Instance: ', err);
+    });
+  // [END bigtable_get_instance]
+}
+
+function getClusters(instanceId) {
+  const instance = bigtableClient.instance(instanceId);
+
+  // [START bigtable_get_clusters]
+  instance
+    .getClusters()
+    .then(result => {
+      console.log(`Clusters: \n${result[0]}`);
+    })
+    .catch(err => {
+      console.error('Error geting Clusters: ', err);
+    });
+  // [END bigtable_get_clusters]
+}
+
+function getAppProfiles(instanceId) {
+  const instance = bigtableClient.instance(instanceId);
+
+  // [START bigtable_get_app_profiles]
+  instance
+    .getAppProfiles()
+    .then(result => {
+      console.log(`AppProfiles: \n${result[0]}`);
+    })
+    .catch(err => {
+      console.error('Error geting AppProfiles: ', err);
+    });
+  // [END bigtable_get_app_profiles]
+}
+
+function getMetaData(instanceId) {
+  const instance = bigtableClient.instance(instanceId);
+
+  // [START bigtable_get_imeta]
```

Comment:
Please use the full name in the tag, specifically, `bigtable_get_instance_metadata`.